crowdsec-docs/docs/appsec/intro.md

```diff
@@ -0,0 +1,46 @@
+---
+id: intro
+title: Introduction
+sidebar_position: 1
+---
+
+# Introduction
+
+<!-- xx : fix crowdsec version -->
+The application security component available in crowdsec:
+
+ - Provides low-effort virtual patching capabilities
+ - Supports your legacy ModSecurity rules
+ - Combines the advantages of classical WAF with the enhanced crowdsec capabilities to allow the detection of behaviors that would otherwise be difficult
+ - Is fully integrated with the Crowdsec software stack, such as the console and remediation components.
+
+<!-- xx :  links -->
+The application security component takes advantage of the remediation components existing in web servers (such as Nginx, Traefik, Haproxy etc.) to provide web application firewall capabilities:
+
+![appsec-global](/img/appsec-global.png)
+
+## Request inspection
+
+1. The remediation component - if the application security component is enabled - forwards the incoming request to the Crowdsec Security engine.
+2. The security engine will first evaluate the inband rules, meant to detect requests that must be blocked. As soon as these rules are processed, an answer is provided to the remediation component:
```

Comment:
[shorten the first part of the sentence a bit, and make the second part introduce the sub-bullet points]

### Security Engine: inband rule processing
The security engine first evaluates the inband rules, designed to identify and block specific requests. <br >
Once these rules are evaluated, a response is relayed to the remediation component. <br >
This leads to two possible outcomes: